agent/app/agent_unspecified.go

```diff
@@ -0,0 +1,23 @@
+// +build !linux
+
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package app
+
+import "golang.org/x/net/context"
```

Comment:
There are a number of places in the Agent code where this happens. I used this here as the dependent package also uses that. We should use `context` everywhere and not this. But, that's outside the context of this PR.